### PR TITLE
Give an example on how to encode base64

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

**Comment by grs** [comment]
@lulf looks like it, we could pin it back to ^3.* for now, I'll push a PR for that

**Comment by grs** [comment]
@lulf it seems to have been triggered by a new major release of the debug library. As a short term fix at least, the following PR should resolve it https://github.com/EnMasseProject/enmasse/pull/1665

File: documentation/common/concept-user-model.adoc [hunk 1]
```diff
@@ -44,6 +44,14 @@ NOTE: At present, `password` type users can be used for both applications and co
 For the `password` type, an additional field `password` must be set to a base64-encoded value of the
 password for that user. The password will not be printed out when reading the resource.
 
+A password can be base64-encoded on the command line, e.g. encoding `my-password`:
```
Comment (jenmalloy) [inline documentation/common/concept-user-model.adoc]: on the command line. To encode `my-password`, for example:

**Comment by jenmalloy** [comment]
@ctron, can you make the requested edit and rebase (so it passes the Travis CI check)? Thanks!

**Comment by ctron** [comment]
> @ctron, can you make the requested edit and rebase (so it passes the Travis CI check)? Thanks!

I though I did that already in be0447c … I re-did it after re-base in the new file.

Is that ok now?

**Pull request merged**